src/quo/components/wallet/token_input/view.cljs

```diff
@@ -14,11 +14,13 @@
     [reagent.core :as reagent]))
 
 (defn calc-value
-  [crypto? currency token value conversion]
+  [crypto? currency token value conversion crypto-decimals]
   (let [num-value (if (string? value) (parse-double (or value "0")) value)]
     (if crypto?
       (str (get common/currency-label currency) (.toFixed (* num-value conversion) 2))
-      (str (.toFixed (/ num-value conversion) 2) " " (string/upper-case (or (clj->js token) ""))))))
+      (str (.toFixed (/ num-value conversion) (or crypto-decimals 2))
```

Comment:
Imo it would be nice to separate the two functions of the if,
e. calc-crypto and calc-fiat 
So it's
(if crypto?
   (calc-crypto ...)
   (calc-fiat ...))
And just reads nicely 👌